trail/upload/process_uploads/fits_processor.py

```diff
@@ -331,8 +331,8 @@ def process(self):
         else:
             standardizedResult.extendThumbnails(tmpResult)
 
-        if len(standardizedResult.thumbnails) != len(standardizedResult.wcs):
-            raise RuntimeError("Can not unambiguously assign thumbnails to WCSs!")
+        # if len(standardizedResult.thumbnails) != len(standardizedResult.wcs):
```

Comment:
Delete these lines. The number of thumbnails does not need to match the number of WCS results in general. I would open a ticket (GH issue or trello) mentioning we don't test with DECAM Community pipeline products and that's why we missed this and that it might be good if we did that.